dev-python/deprecation/deprecation-2.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 python3_5 python3_6 )
+inherit distutils-r1
+
+DESCRIPTION="A library to handle automated deprecations"
+HOMEPAGE="https://deprecation.readthedocs.io/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 x86"
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+
+RDEPEND="
+	dev-python/packaging[${PYTHON_USEDEP}]
+"
```

Comment:
Please include running the tests. You need to define `python_test()` phase (relevant docs on https://wiki.gentoo.org/wiki/Project:Python + other ebuilds). If tests are not included in pypi tarball, please use snapshot from github instead.